### ISSUE TITLE
can u show us an example or tutorial how to use ?

### ISSUE THREAD

**Issue by 2021php** [Podiatry30259/KeyAuth-Bypass#1]
can u show us an example or tutorial how to use ?

**Comment by LiamG53** [comment]
works good thank you :) however my merge branch will prevent this from working if u call that orignal function 🛹

**Comment by Podiatry30259** [comment]
yes in order to get this working just place a hook on the address provided.
**this is not patched still and wont ever be you can dynamically find the class instance and get the names, secret id etc.. this is unpatchable.**
